src/RA_httpthread.cpp

```diff
@@ -245,8 +245,8 @@ BOOL DoBlockingImageUpload(UploadType nType, const std::string& sFilename, std::
     // Use WinHttpOpen to obtain a session handle.
     HINTERNET hSession = WinHttpOpen(RAWeb::GetUserAgent().c_str(),
         WINHTTP_ACCESS_TYPE_DEFAULT_PROXY,
-        WINHTTP_NO_PROXY_NAME,
-        WINHTTP_NO_PROXY_BYPASS, 0);
+        nullptr,
+        nullptr, 0);
```

Comment:
Why suppress for the LVIR constants, but not for the WINHTTP constants? The [documentation](https://docs.microsoft.com/en-us/windows/desktop/api/winhttp/nf-winhttp-winhttpopen) specifically says the constants must be used:
> If dwAccessType is not set to WINHTTP_ACCESS_TYPE_NAMED_PROXY, this parameter must be set to WINHTTP_NO_PROXY_NAME.

Reply:
Same comment applies several times below.

Reply:
`LVIR_BOUNDS` parameter is an `int` but the macro is defined like this:
```
#define ListView_GetSubItemRect(hwnd, iItem, iSubItem, code, prc) \
        (BOOL)SNDMSG((hwnd), LVM_GETSUBITEMRECT, (WPARAM)(int)(iItem), \
                ((prc) ? ((((LPRECT)(prc))->top = (iSubItem)), (((LPRECT)(prc))->left = (code)), (LPARAM)(prc)) : (LPARAM)(LPRECT)NULL))
```

I changed the `WINHTTP` constants (even though they are pointers) as requested anyway.

Reply:
If it were C, `NULL` would be `(void*)0` which is the same as `nullptr` in C++. You can't implicitly cast `void*` to pointer to object so it's `0` in C++, that's why there's `nullptr`.